app/services/analytics.rb

```diff
@@ -123,8 +123,6 @@ def browser_attributes
   ACCOUNT_DELETE_VISITED = 'Account Delete visited'.freeze
   ACCOUNT_DELETION = 'Account Deletion Requested'.freeze
   ACCOUNT_RESET_VISIT = 'Account deletion and reset visited'.freeze
-  ACCOUNT_VERIFICATION_SUBMITTED = 'Account verification submitted'
-  ACCOUNT_VERIFICATION_VISITED = 'Account verification visited'
```

Comment:
do we care about querying historical data consistently? if so, I would recommend renaming just the constant and not the value

Reply:
> do we care about querying historical data consistently? if so, I would recommend renaming just the constant and not the value

I think one of the main goals of this work is to make the logged event name a bit more sensible, so changing the value would be part of that. Per our [recent discussion around logging backward-compatibility](https://docs.google.com/document/d/1g_V2vlT79tScBKIVw7M4UXf15TrG69iUrLHE0yE1GGI/edit#bookmark=id.8h5cmbvtlfxm), I had checked to see if these were included in any `identity-devops` Terraform'd queries, and wasn't able to find anything.

Reply:
Yup, just meant it we wanted to spot check or run an adhoc query.

Maybe we could put ` # previously: "Account verification submitted" ` or something next to the new lines?

Reply:
> Maybe we could put `# previously: "Account verification submitted"` or something next to the new lines?

Sure, that seems reasonable enough 👍

Reply:
Updated in f0f98f9b2.